Makefile

```diff
@@ -156,7 +156,7 @@ format: $(GOIMPORTS)
 # Generate golden files used in unit tests
 .PHONY: gen-test-golden
 gen-test-golden:
-gen-test-golden: PKG = ./cmd/helm ./pkg/action
+gen-test-golden: PKG = ./pkg/cmd ./pkg/action
```

Comment:
Note, this change should have happened when the cmd functionality was moved to pkg so that it could be reused. This is part of a make script that can update golden files en masse.